### PR TITLE
fix(ci): set LOTUS_SYNC_BOOTSTRAP_PEERS=1 to let lotus daemon sync

### PR DESCRIPTION
**Pull request by hanabi1224**
## Summary of changes

This PR fixes an issue that CI jobs that run lotus daemon get stuck due to bootstrap peer connectivity issue

Changes introduced in this pull request:

-

## Reference issue to close (if applicable)

Closes

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

**Comment by LesnyRumcajs** [comment]
Note that this is a temporary workaround if we intend to have Forest as bootstrap nodes. The root issue was that the Glif node was unreachable for some, and then even more ChainSafe-hosted Lotus bootstrap nodes were not working properly. The only bootstrapper, for some regions, that was pingable was Forest, but it was not able to bootstrap other Lotus peers. They would connect to it, even exchange some blocks, but new peers would not get propagated.

**Review by LesnyRumcajs — APPROVED** [review]

File: scripts/devnet/.env [hunk 1]
```diff
@@ -2,6 +2,7 @@ LOTUS_IMAGE=ghcr.io/chainsafe/lotus-devnet:2024-07-03-1f030c2
 FOREST_DATA_DIR=/forest_data
 LOTUS_DATA_DIR=/lotus_data
 FIL_PROOFS_PARAMETER_CACHE=/var/tmp/filecoin-proof-parameters
+LOTUS_SYNC_BOOTSTRAP_PEERS=1
```
Comment (LesnyRumcajs) [inline scripts/devnet/.env:5]: there's no need to do it here, 2k networks have this value at `1` by default.

Comment (hanabi1224) [reply scripts/devnet/.env:5]: Yes but it doesn't hurt right?

Comment (LesnyRumcajs) [reply scripts/devnet/.env:5]: It looks like we are overriding a default behaviour (but we don't), it doesn't bring any value in this context, while only increasing the complexity if anyone is analysing it. That said, it's just a personal preference, not a huge deal.

**Review by lemmih — APPROVED** [review]

**Pull request merged**